forecasting/src/autogluon/forecasting/task/forecasting/predictor.py

```diff
@@ -0,0 +1,182 @@
+import copy
+import logging
+
+import pandas as pd
+
+from .dataset import TimeSeriesDataset
+from core.task.base import BasePredictor
+from core.utils import plot_performance_vs_trials, plot_summary_of_models, plot_tabular_models, verbosity2loglevel
```

Comment:
These imports need to include `autogluon` since `autogluon.core` is a separate namespace package.  See for example:  https://github.com/awslabs/autogluon/blob/master/tabular/src/autogluon/tabular/task/tabular_prediction/dataset.py#L5

Reply:
changed.